### DISCUSSION TITLE
Links are not crawlable issues in LightHouse report

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#5470] [Q&A] [answered]
For blog pages on our site, when I generate lighthouse report - it gives low score on SEO for our blog citing `Links are not crawlable` issue (attached screenshot) 

Though this seems to be something which is added by docusaurus by default - and not anything specific about our blog. 

Is there a way to solve this so that our SEO performace improves? ( though not very sure why this should impact SEO performance)

This is the blog I am trying to optimise - https://signoz.io/blog/opentelemetry-vs-prometheus/

![Screenshot 2021-09-02 at 7 47 05 PM](https://user-images.githubusercontent.com/504541/131860328-7ed90626-f5b4-471e-9e36-5892d1bab6ed.png)

**Comment by slorber** [comment]
We seem to have the same report on our own blog.

https://webmasters.stackexchange.com/questions/132481/lighthouse-says-links-are-not-crawlable-on-the-skip-to-content-link

Not sure but we may need to replace TOC links like `<a id="xyz"/>` by something else, like a span or something 🤷

**Comment by slorber** [comment, marked as answer]
fixed in https://github.com/facebook/docusaurus/pull/5481

**Comment by pranay01** [comment]
Awesome, thanks. Will wait for this to be pushed in a release
